t/34-monitor-runfolder-staging.t

```diff
@@ -99,42 +96,39 @@ subtest 'updating run data from filesystem' => sub {
       expected_cycle_count => 310,
       actual_cycle_count => 0,
```

Comment:
Just to make sure we have setup thing for following tests and rely less on the yml fixture loading
```suggestion
      actual_cycle_count => 0,
      folder_name => undef,
      folder_path_glob => undef,
```